<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="loginPage">
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>共享账本</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signinPage">
    <widget class="QLineEdit" name="pwEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>61</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="signButton_1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retnButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwconEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
